src/components/root/styled.ts

```diff
@@ -1,14 +1,10 @@
 import styled from 'styled-components';
 
-import Common from '../common/styled';
-
-const Root = styled(Common.Container)`
+const Root = styled.div`
```

Comment:
Hvorfor ble inheritance fjernet her?

Reply:
Container setter 'max-width', mens det vil vi nå kun ha på innholdet i midten (DataServicePage), ikke for header og footer som skal gå fullt ut i bredden.